runtime/mandala/src/lib.rs

```diff
@@ -1974,30 +1978,36 @@ impl Convert<(Call, SignedExtra), Result<EthereumTransactionMessage, InvalidTran
 					return Err(InvalidTransaction::Stale);
 				}
 
-				let era: frame_system::CheckEra<Runtime> = extra.3;
+				let era: frame_system::CheckEra<Runtime> = extra.clone().3;
```

Comment:
```suggestion
				let era: frame_system::CheckEra<Runtime> = extra.3.clone();
```
Not 100% sure this works, but should have a way to only do one clone instead one per accessed field.

Otherwise could be 

`let (_, _, _, era, check_nonce, _, tip) = extra.clone();